src/components/DayOverview/DayOverview.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import supabase, { Schema } from "@/supabase";
+import DayOverviewButton from "@/components/DayOverview/DayOverviewButton";
+
+interface Props {
+    text: string;
+}
+
+export type ParcelOfSpecificDateAndLocation = {
+    collection_datetime: Schema["parcels"]["collection_datetime"];
+    clients: Pick<
+        Schema["clients"],
+        "flagged_for_attention" | "full_name" | "address_postcode" | "delivery_instructions"
+    > | null;
+};
+
+const getParcelsOfSpecificDateAndLocation = async (
+    date: Date,
+    collectionCentre: string
+): Promise<ParcelOfSpecificDateAndLocation[]> => {
+    const startDateString = date.toISOString();
+    const endDate = new Date(date);
+    endDate.setDate(date.getDate() + 1);
+    const endDateString = endDate.toISOString();
+
+    const { data, error } = await supabase
+        .from("parcels")
+        .select(
+            "collection_datetime, clients(flagged_for_attention, full_name, address_postcode, delivery_instructions)"
+        )
+        .gte("collection_datetime", startDateString)
+        .lt("collection_datetime", endDateString)
+        .eq("collection_centre", collectionCentre)
+        .order("collection_datetime");
+
+    if (error) {
+        throw new Error(`Database error: ${JSON.stringify(error)}`);
+    }
+
+    return data;
+};
+
+const DayOverview = async ({ text }: Props): Promise<React.ReactElement> => {
+    const date = new Date("2023-07-17");
```

Comment:
Seems odd to not have this passed in as part of the props as opposed to being hard coded. In any use case people would choose the date from the page, not here.

Reply:
Oh wait nvm if you don't have a page that is fine. However, what is this page used for? Like is it a dummy page? In which case it should probably not exist when merging and the interfaces etc should be contained in the DayOverviewButton File

Reply:
Oh wait actually I realise you and @paopaosoftwire are right. I should move it to the dummy `/pdfDownloader` page instead because `DayOverview` is not a dummy page - as `DayOverviewButton` is on client side but data fetching should be done on the async side